File: 1 - Iniciante/C99/BEE1134.c
#include <stdio.h>

int main(void){
	int tipo=0;
	int alcool=0, gasolina=0, diesel=0;
	
	while (tipo != 4){
	
		scanf("%d", &tipo);	
		switch (tipo){
			case 1:
				alcool++;
				break;
			case 2:
				gasolina++;
				break;
			case 3:
				diesel++;
				break;
		}
		
	}
	
	printf("MUITO OBRIGADO\n");
	printf("Alcool: %d\n", alcool);
	printf("Gasolina: %d\n", gasolina);
	printf("Diesel: %d\n", diesel);
	
	return 0;
}

File: 1 - Iniciante/Python 3.11/BEE1134.py
alcool, gasolina, diesel = 0, 0, 0
entrada = 0
while (entrada != 4):
    entrada = int(input())
    match (entrada):
        case 1:
            alcool += 1
        case 2:
            gasolina += 1
        case 3:
            diesel += 1
        case 4:
            break
        case _:
            continue

print("MUITO OBRIGADO")
print(f'Alcool: {alcool}')
print(f'Gasolina: {gasolina}')
print(f'Diesel: {diesel}')